src/main/transports/net.ts

```diff
@@ -2,17 +2,34 @@ import { Transports } from '@sentry/node';
 import { Event, Response, SentryRequest, Status, TransportOptions } from '@sentry/types';
 import { parseRetryAfterHeader, PromiseBuffer, SentryError, timestampWithMs } from '@sentry/utils';
 import { net } from 'electron';
+import { Readable, Writable } from 'stream';
 import * as url from 'url';
+import { createGzip } from 'zlib';
 
 import { isAppReady } from '../backend';
 
+const GZIP_THRESHOLD = 1024 * 32;
```

Comment:
_nit_: I'd mention in a comment that this is an estimated maximum size for reasonable standalone events.